ray-operator/controllers/ray/raycluster_controller_unit_test.go

```diff
@@ -3007,111 +3008,6 @@ func Test_RedisCleanup(t *testing.T) {
 	}
 }
 
-func Test_RedisCleanupWithLoginShell(t *testing.T) {
```

Comment:
TestGetContainerCommand already tests the flag. It is not worth it to maintain this complex test, so I move the check to `Test_RedisCleanup`.

We should refactor the interface to make write unit tests easier.